packages/features/ee/workflows/components/WorkflowStepContainer.tsx

```diff
@@ -919,32 +945,9 @@ export default function WorkflowStepContainer(props: WorkflowStepProps) {
                       )}
                   </div>
                 )}
-
-                {step.action !== WorkflowActions.SMS_ATTENDEE &&
-                step.action !== WorkflowActions.SMS_NUMBER ? (
-                  <>
-                    <div className="mb-2 flex items-center pb-[1.5px]">
-                      <Label className="mb-0 flex-none ">
-                        {isEmailSubjectNeeded ? t("email_body") : t("text_message")}
-                      </Label>
-                    </div>
-                    <Editor
-                      getText={() => {
-                        return props.form.getValues(`steps.${step.stepNumber - 1}.reminderBody`) || "";
-                      }}
-                      setText={(text: string) => {
-                        props.form.setValue(`steps.${step.stepNumber - 1}.reminderBody`, text);
-                        props.form.clearErrors();
-                      }}
-                      variables={DYNAMIC_TEXT_VARIABLES}
-                      height="200px"
-                      updateTemplate={updateTemplate}
-                      firstRender={firstRender}
-                      setFirstRender={setFirstRender}
-                      editable={!props.readOnly && !isWhatsappAction(step.action)}
-                    />
-                  </>
-                ) : (
+                {!isEmailSubjectNeeded && !isSenderIsNeeded && <editorComponent isEditable={false} />}
+                {isEmailSubjectNeeded && <editorComponent isEditable={true} />}
+                {isSenderIsNeeded && (
```

Comment:
Seems to me like an issue with the Editor Component, you can render it the way it is done in other places


https://github.com/calcom/cal.com/blob/b09b32b121adb50f903bcfdadf62a39ae1a6f68b/apps/web/components/eventtype/EventSetupTab.tsx#L114

https://github.com/calcom/cal.com/blob/b09b32b121adb50f903bcfdadf62a39ae1a6f68b/apps/web/components/eventtype/EventSetupTab.tsx#L28-L45